contracts/Colony.sol

```diff
@@ -277,6 +281,15 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     emit ColonyUpgraded(currentVersion, _newVersion);
   }
 
+  function finishUpgrade2To3() public always {
+    ColonyAuthority(address(authority)).setRoleCapability(
+      uint8(ColonyDataTypes.ColonyRole.Root),
+      address(this),
+      bytes4(keccak256("updateColonyOrbitDB(string)")),
+      true
+    );
+  }
+
```

Comment:
Again for the `upgrade` function we should probably limit that to sequential version upgrades only.